core/server/master/src/test/java/alluxio/master/file/FileSystemMasterFsOptsTest.java

```diff
@@ -105,6 +105,37 @@ public void createFileUsesOperationTime() throws Exception {
     assertEquals(100, info.getLastAccessTimeMs());
   }
 
+  @Test
+  public void createFileWithOverwrite() throws Exception {
+    AlluxioURI path = new AlluxioURI("/test");
+    mFileSystemMaster.createFile(path, CreateFileContext.defaults());
+    //create without overwrite
+    try {
+      mFileSystemMaster.createFile(path, CreateFileContext.defaults());
```

Comment:
nit: a better style is
```suggestion
    Exception e = assertThrows(FileAlreadyExistsException.class, () -> {
          mFileSystemMaster.createFile(path, CreateFileContext.defaults());
    });
    assertTrue(e.getMessage().contains(...));
```

Reply:
it is just the error is easier to understand when the code does not behave your way (does not throw the right ex)

Reply:
done